tests/e2e/logging/test_openshift-logging.py

```diff
@@ -171,3 +177,60 @@ def test_create_new_project_to_verify_logging(self, create_pvc_and_deploymentcon
         pod_obj.run_io(storage_type='fs', size=6000)
 
         self.validate_project_exists(pvc_obj)
+
+    @pytest.mark.polarion_id("OCS-650")
+    @tier4
+    def test_respin_osd_pods_to_verify_logging(self, create_pvc_and_deploymentconfig_pod):
+        """
+
+        """
+        # Create 1st project and app_pod
+        dc_pod_obj, dc_pvc_obj = create_pvc_and_deploymentconfig_pod
+
+        project1 = dc_pvc_obj.project.namespace
+
+        # Delete the OSD pod
+        disruption = disruption_helpers.Disruptions()
+        disruption.set_resource(resource='osd')
+        disruption.delete_resource()
+
+        # Check the health of the cluster-logging
+        assert ocp_logging_obj.check_health_of_clusterlogging()
+
+        # Check for the 1st project created in EFK stack after respin
+        pod_list = get_all_pods(namespace='openshift-logging')
+        elasticsearch_pod = [
```

Comment:
I see this operation of getting all elasticsearch pods is done in multiple test cases. It will be a good idea to put it into a function. Probably in a subsequent PR.

Reply:
Done